mscore/palette.cpp

```diff
@@ -763,6 +763,16 @@ void Palette::mouseDoubleClickEvent(QMouseEvent* ev)
             return;
       if (score->selection().isNone())
             return;
+      
+      // exit edit mode, to allow for palette element to be applied properly
+      ScoreView* viewer = mscore->currentScoreView();
+      if (viewer && viewer->editMode() && mscore->currentScoreView()) {
+            ScoreState scoreState = mscore->currentScoreView()->mscoreState();
```

Comment:
why not use viewer instead of mscore->currentScoreView()?

Reply:
gosh, I was being redundnat there, my bad...